src/kfactory/kcell.py

```diff
@@ -1635,9 +1725,12 @@ def ghost_cell(self, value: bool) -> None:
             raise LockedError(self)
         self._kdb_cell.ghost_cell = value
 
-    def __getattr__(self, name):  # type: ignore[no-untyped-def]
+    def __getattr__(self, name: str) -> Any:
         """If KCell doesn't have an attribute, look in the KLayout Cell."""
-        return getattr(self._kdb_cell, name)
+        try:
+            return super().__getattr__(name)  # type: ignore
+        except Exception:
+            return getattr(self._kdb_cell, name)
```

Comment:
Are you sure this is necessary? If not, I would like to keep it as the original way. Exception handling is not super cheap (also not that expensive in the python context, but still saves 4-5 sec on the tests).

Reply:
Recursion error as when you call self._kdb_cell, since its a private attr, it calls self.__getattr__("_kdb_cell"), so you need to call super().__getattr__ to get the cell, and if you do self.shapes for example it will fail the super get attr and do the _kdb_cell get attr

Reply:
I see, then maybe we make a match/ifelse for kdb_cell?

Reply:
Oh, I see, that might not work. Hmm, then let's keep it like this for now